index.html

```diff
@@ -0,0 +1,341 @@
+				of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time
+				and space and energy to science and speculation.</p>
+
+		</div>
+
+		<div id="aside-one">
+			<h5>Editors' Picks</h5>
+			<div class="aside-one-row">
+				<img src="images/A1.png" alt="space">
+				<a
+					href="https://www.nytimes.com/2019/01/07/world/asia/philippines-manila-jail-overcrowding.html?fallback=0&recId=1G4SPPsUZko1MRIdTiNirqBx4mO&locked=0&geoContinent=NA&geoRegion=AB&recAlloc=story-desks&geoCountry=CA&blockId=signature-journalism-vi&imp_id=169928447">Where
+					518 Inmates Sleep in Space for 170, and Gangs Hold It Together</a>
+			</div>
+			<div class="aside-one-row">
+				<img src="images/a2.png" alt="pizza">
+				<a
+					href="https://www.nytimes.com/2019/01/16/dining/slice-joint-pizza-new-york-city.html?fallback=0&recId=1G4SPPsUZko1MRIdTiNirqBx4mO&locked=0&geoContinent=NA&geoRegion=AB&recAlloc=story-desks&geoCountry=CA&blockId=signature-journalism-vi&imp_id=797101841">How
+					the Slice Joint Made Pizza the Perfect New York City Food</a>
+			</div>
+			<div class="aside-one-row">
+				<img src="images/a3.png" alt="church">
+				<a
+					href="https://www.nytimes.com/2019/01/11/nyregion/new-utrecht-reformed-church.html?fallback=0&recId=1G4SPPsUZko1MRIdTiNirqBx4mO&locked=0&geoContinent=NA&geoRegion=AB&recAlloc=story-desks&geoCountry=CA&blockId=signature-journalism-vi&imp_id=673155223">Can
+					a Church Founded in 1677 Survive the 21st Century?</a>
+			</div>
+		</div>
+
+		<div class="article-segment-two">
+
+			<p>Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with
+				its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent,
+				architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless number
+				of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.</p>
+
+			<h3>‘As Big as It Gets’</h3>
+
+			<p>In our own universe, it would serve as a window into the forces operating at energies forever beyond the
+				reach of particle accelerators on Earth and yield new insights into gravity itself. Dr. Kovac’s ripples
+				would be the first direct observation of gravitational waves, which, according to Einstein’s theory of
+				general relativity, should ruffle space-time.</p>
+
+			<p>Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part of the team,
+				said, “This is huge, as big as it gets.”</p>
+
+			<p>He continued, “This is a signal from the very earliest universe, sending a telegram encoded in
+				gravitational waves.”</p>
+
+		</div>
+
+		<div id="aside-two">
+			<h5>Sign up for Science Times</h5>
+			<p>We’ll bring you stories that capture the wonders of the human body, nature and the cosmos.</p>
+			<input type="button" value="SIGN UP">
+		</div>
```

Comment:
- [ ]  Consider rephrasing the comments as suggestions instead of orders. For example, instead of "Use aside tag", you can give a polite suggestion as "I think it would be better to use aside tags here".
- [ ] Consider adding an explanation for each comment.  For example, explain why it is better to use an aside tag. In this way, your suggestions are valuable feedback.
- [ ] Consider giving more details to your comments. For example, you commented, "The Sign up for Science Times section does not display properly". Why don't you add how should it be displayed?